std::vector is passed to this, which is not guaranteed to be printable

--- a/testsuite/backend/genesys/minigtest.h
+++ b/testsuite/backend/genesys/minigtest.h
@@ -40,7 +40,8 @@ void check_equal(const T& t, const U& u, const char* function, const char* path,
         s_num_failures++;
         std::cerr << "FAILURE at ";
         print_location(std::cerr, function, path, line);
-        std::cerr << " :\n" << t << " != " << u << "\n\n";
+        //std::cerr << " :\n" << t << " != " << u << "\n\n";
+        std::cerr << "\n";
     } else {
         s_num_successes++;
         std::cerr << "SUCCESS at ";
